scene/gui/rich_text_label.cpp

```diff
@@ -5779,6 +5780,8 @@ void RichTextLabel::_bind_methods() {
 
 	ADD_SIGNAL(MethodInfo("finished"));
 
+	ADD_SIGNAL(MethodInfo("thread_finish"));
```

Comment:
This needs to be documented, run `godot --doctool`

Reply:
Thank you for you review. I will try it.